template/package.json

```diff
@@ -133,11 +133,11 @@
     "build:ci": "npm-run-all sweep build",
     "build:api": "wca analyze 'src/[namespace]-[name].js' --outFiles docs/api.md",
     "build:demo": "npm-run-all build demo:rm:build demo:new:build demo:copy:index demo:copy:demo demo:update:index",
-    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render build:docs",
     "build:docs": "node scripts/generateDocs.js",
     "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
     "build:sass:component": "sass --no-source-map src:src",
-    "build:watch": "nodemon -e scss,js --watch src --exec npm run build:dev:assets",
+    "build:watch": "nodemon -e scss,js,html --watch src --watch apiExamples/**/* --exec npm run build:dev:assets",
```

Comment:
Given the recent changes to combobox to make the examples work - 

Perhaps we can run the bundler when JS files are added to or changed in apiExamples. If so, we can make localhost point to the minimized JS files which gives us a little more certainty that the examples will work on the doc site correctly.

Reply:
I am open to that idea, let's create a new issue to discuss this as an option to add as a feature post this update.